File: kits/go/myBot.go
package main

import (
	"fmt"
	"./kit"
)

func main() {
		// create a new agent
		agent := kit.Agent {}
		// intialize agent
		agent.Initialize();

		for true {
				// wait for update from match engine
				agent.Update();
				// send some commands
				fmt.Printf("send some command, another command\n");
				// end turn
				agent.EndTurn();
				
		}
}

File: kits/c++/myBot.cpp

#include "kit.hpp"
#include <string>

int main() {
  kit::Agent agent = kit::Agent();
  // initialize
  agent.initialize();

  while(true) {
    // wait for updates
    agent.update();
    // send some commands
    std::cout << 'my command, other command' << std::endl << std::flush;
    // end turn
    agent.end_turn();
    
  }
  
  return 0;
}